src/main/kotlin/oncall/dto/ScheduledResponse.kt

```diff
@@ -0,0 +1,19 @@
+package oncall.dto
+
+import oncall.domain.calendar.CustomDayOfWeek
+import oncall.domain.calendar.Day
+import oncall.domain.member.Member
+
+class ScheduledResponse(member: Member, day: Day) {
```

Comment:
Maybe we can make a factory method in the companion object.
Then we can declare this DTO as a data class.

```suggestion
data class ScheduledResponse(
    val dayOfWeekAsText: String,
    val note: String,
    val name: String,
) {
    companion object {
        private fun generateLegalHolidayFormat(day: Day): String {
            if (day.isLegalHoliday) {
                return "(휴일)"
            }
            return ""
        }
        fun of(member: Member, day: Day): ScheduledResponse {
            val dayOfWeekAsText = CustomDayOfWeek.from(day.dayOfWeek).text
            val note = generateLegalHolidayFormat(day)
            val name = member.name
            return ScheduledResponse(dayOfWeekAsText, note, name)
        }
    }
}
```